panther_manager/include/panther_manager/plugins/action/call_set_bool_service_node.hpp

```diff
@@ -8,7 +8,7 @@
 namespace panther_manager
 {
 
-class CallSetBoolService : public panther_manager::RosServiceNode<std_srvs::SetBool>
+class CallSetBoolService : public RosServiceNode<std_srvs::SetBool>
 {
 public:
```

Comment:
If the parent class will be updated with the ports this node can be reduced to header only